pkg/secrets/secrets.go

```diff
@@ -52,6 +52,23 @@ func GenerateUserData(publicKey ssh.PublicKey) (*core.Secret, error) {
 		},
 		Data: map[string][]byte{
 			"userData": []byte(`<powershell>
+			function Get-RandomPassword {
+				Add-Type -AssemblyName 'System.Web'
+				return [System.Web.Security.Membership]::GeneratePassword(16, 2)
+			}
+
+			# Check if the capi user exists, this will be the case on Azure, and will be used instead of Administrator
+			if((Get-LocalUser |Where-Object {$_.Name -eq "capi"})  -eq $null) {
```

Comment:
nits: please put a space after the pipe and remove the extra space before `-eq` (there's 2)